docs/source/references/apollo-engine.md

```diff
@@ -0,0 +1,158 @@
+
+The Engine GitHub application asks for permission to read which GitHub organizations you’re in and their members and teams (but not code!). If you grant Engine permission to see an organization, we create an Engine account with the same name as that GitHub organization. All members of that organization on GitHub will be able to see the new account in Engine. This is how you create a shared team account in Engine.
+
+When you sign in to Engine, you will have access to all the teams where you're a member of the organization on GitHub. You can use the organization account picker to switch between accounts. If another member of a GitHub organization you belong to has already signed up the GitHub organization for Engine access, you’ll have access to that existing account.
+
+If you’d like to work with additional team members and you are the admin of a GitHub organization, simply add them to your GitHub organization. If you aren’t an admin, have an admin add you to their GitHub organization.
+
+<h3 id="add-organization">Adding an organization</h3>
+
+If you’re looking for a GitHub organization that you’re a member of and don’t see it in Engine, it’s likely that Engine does not have read access for that organization.
+
+If you want to Add or Remove an organization from Engine, you should manage those settings on GitHub. There, you will be able to Grant or Revoke access to Engine for organizations you can administer. For organizations you do not administer, you can Request access to Engine and the administrators will receive a request by E-mail.
+
+<h3 id="github-permissions">GitHub permissions</h3>
+
+GitHub’s OAuth service is used for read-only information about organizations and users. Engine does not need access rights to your source code or to any other sensitive data in its login system.
+
+If your Engine account is owned by a GitHub organization, then Engine will allow all members of that organization to access the account. As you add or remove team members from your Github org, Engine will know about that and accordingly update the authorization for those users.
+
+<h3 id="other-logins">Support for non-GitHub login</h3>
+
+Engine was built to mirror GitHub’s login and does not currently support other forms of login or team creation. Our current recommendation for teams who want to use Engine but do not use GitHub is:
+
+1. Create a free GitHub organization (empty, public).
+2. Add your desired team members to the organization.
+3. Hide membership of the organization if desired.
+4. Use this organization to collaborate on Engine.
+
+
+<h2 id="services">Services</h2>
+
+A service in Engine is an entity that is provisioned in the Engine cloud service that your server can report performance metrics and schema versions to via an API key. The information reported by your server is processed and stored in the Engine cloud service and made accessible to you through the Engine interface.
```

Comment:
The relationship between a service and your GraphQL API is unclear from this paragraph. Is it always a 1:1 mapping?